### DISCUSSION TITLE
Trade data EU

### DISCUSSION THREAD

**Discussion by Shhemant** [Depart-de-Sentier/sentier.dev#36] [Data suggestion]
### Data Source name

Access2markets

### Data Source Homepage URL

https://trade.ec.europa.eu/access-to-markets/en/home

### Data License

Free, publicly available data

### Justify this Data Source Choice

Not sure how exactly is the entire database available. It contains import & export data for different consumer products like clothing, footwear, and so on.

### Area of Application

Information available is:
- Number, weight and price of goods imported or exported from each EU country
- Approximate product composition of these goods (e.g. cotton shirts)

can help in creating averages for different products

**Comment by oliver-hurtig** [comment]
I believe this might be the same data as from Eurostat:

![image](https://github.com/user-attachments/assets/a579eaae-93c1-4205-aa93-1a948e318305)

Eurostat has an API and a lot of other data on production, consumption, trade, electricity, etc.
